modules/cluster-resources.adoc

```diff
@@ -0,0 +1,39 @@
+[id="support-cluster-resources_{context}"]
+= Interacting with your cluster resources
+
+You can interact with cluster resources by using the OpenShift CLI (`oc`) tool in {product-title}. The cluster resources that you see after running the `oc api-resources` command can be edited.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the web console or you have installed the `oc` CLI tool.
+
+.Procedure
+
+. To see which configuration operators have been applied, run the following command:
```

Comment:
Is this "Operators" in the [OpenShift](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization) context?
```suggestion
. To see which configuration Operators have been applied, run the following command:
```

Reply:
Yes. In the Openshift context

Reply:
acked